composer.json

```diff
@@ -28,7 +27,8 @@
         "zendframework/zend-diactoros": "~1.0"
     },
     "suggest": {
-        "ext-openssl": "To use Security::encrypt() or have secure CSRF token generation."
+        "ext-openssl": "To use Security::encrypt() or have secure CSRF token generation.",
+        "lib-ICU": "The ICU PHP library."
```

Comment:
I think it is only used in `Text::transliterate()` and therefore in `Text::slug()`